packages/web/test/mocks/myalgowallet-mock.ts

```diff
@@ -0,0 +1,91 @@
+import type {
+    Options,
+    Accounts,
+    Address,
+    AlgorandTxn,
+    Base64,
+    ConnectionSettings,
+    EncodedTransaction,
+    SignedTx,
+    SignTransactionOptions,
+} from "@randlabs/myalgo-connect";
+
+export default class MyAlgoConnectMock {
+
+    /**
+     * @param {Options} options Override default popup options.
+     */
+    constructor(options?: Options) {
+
+    };
+
+    /**
+     * @async
+     * @description Receives user's accounts from MyAlgo.
+     * @param {ConnectionSettings} [settings] Connection settings
+     * @returns Returns an array of Algorand addresses.
+     */
+    connect(settings?: ConnectionSettings): Promise<Accounts[]> {
```

Comment:
You are returning empty. Y???

Reply:
we need to initialize it, not necessarily to return any values

Reply:
Then how about changing the function name to `init(..)`?

Reply:
actually can't rename the method since it needs to be in sync with the MyAlgoWallet class, since it is mock it needs to be a copy of it